2996_reflection/reflection.md

```diff
@@ -5124,7 +5097,7 @@ Change paragraph 2 of [dcl.fct.def.delete]{.sref} to allow for reflections of de
 
 ::: std
 
-[2]{.pnum} A program that refers to a deleted function implicitly or explicitly, other than to declare it [or to use as the operand of a `$reflect-expression$` ([expr.reflect])]{.addu}, is ill-formed.
+[2]{.pnum} A program that refers to a deleted function implicitly or explicitly, other than to declare it [or to appear as the operand of a `$reflect-expression$` ([expr.reflect])]{.addu}, is ill-formed.
```

Comment:
Should this talk about being designated instead of appearing?

Reply:
I don't think so, because the operand to the _reflect-expression_ is always a name. So I think "appear" is fine, and seems consistent with some other wording I encountered elsewhere.